docs/reference/server-configuration.md

```diff
@@ -48,6 +48,14 @@ to, set `ksql.advertised.listener` to a URL that ksqlDB nodes can resolve.
 
 For more information, see [Configuring Listeners of a ksqlDB Cluster](./index.md#configuring-listeners-of-a-ksqldb-cluster)
 
+## `ksql.compression.type`
+
+Sets the compression type used by {{ site.ak }} producers, like the
+INSERT VALUES statement. This setting is distinct from the
+`ksql.streams.producer.compression.type` config, which sets the type of 
+compression used by streams producers for topics created by CREATE TABLE AS
+SELECT and CREATE STREAM AS SELECT statements.
```

Comment:
Should `INSERT INTO` also be mentioned as a type of query where `ksql.streams.producer.compression.type` applies? I don't feel strongly about it, especially since the difference between `INSERT VALUES` and `INSERT INTO` might not be obvious to all users.

Reply:
Also, should the default (`none`?) be mentioned here?

Reply:
Good thinking @mikebin! `INSERT INTO` and `INSERT VALUES` is very subtle.

Btw: we also set the default to `snappy` in cloud -- there is already a PR. Not sure how quickly it will be rolled out though -- we could also do a follow up PR after the default change was rolled out to cloud?

Reply:
@mikebin @mjsax kk I added INSERT INTO to the `ksql.streams.producer.compression.type` section, and mentioned that the default is `none` for both configs. I'll open a ticket to track the change from `none` to `snappy` for `ksql.compression.type`.

Reply:
I think the default is `snappy` for `ksql.streams.producer.compression.type`

Reply:
@mikebin cool thanks, does that mean the note in the `ksql.streams.producer.compression.type` is unnecessary? "In Confluent Cloud ksqlDB, the default is `snappy`."

Reply:
I think `ksql.streams.producer.compression.type=snappy` by default for both Cloud and on-prem. On the other hand, I think `ksql.compression.type=none` by default, at least until the PR @mjsax mentioned goes in.

Reply:
Cool, thanks for the feedback, @mikebin and @mjsax!

Reply:
Not sure about on-prem, but for cloud it's currently:
```
ksql.streams.producer.compression.type=snappy
ksql.compression.type=none
```
And we will change could to `snappy` for both.